salt/scripts.py

```diff
@@ -46,9 +46,24 @@ def _handle_signals(client, signum, sigframe):
         hardcrash = client.options.hard_crash
     except (AttributeError, KeyError):
         hardcrash = False
+
+    exit_msg = '\nExiting gracefully on Ctrl-c'
```

Comment:
/me shrugs

I didn't change that part of it, this was existing behavior merged in [#35923](https://github.com/saltstack/salt/pull/35923/files). Is the ``SystemExit`` being caught by the ``KeyboardInterrupt`` in ``salt/client/__init__.py`` what is generating this SIGTERM?